2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,9 +18,15 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let countryCurCodes = {};
+  countryCurrencyCodes.map((arr) => {
```

Comment:
This code works, but what is the purpose of `Array.map`?

Reply:
Yes, we don't need to return a new array. forEach method fits better here

Reply:
Also, `Array.reduce`:
```
let countryCurCodes = countryCurrencyCodes.reduce((curCodesMap, curCodesArray) => {
    return {
        ...curCodesMap,
        [curCodesArray[0]]: curCodesArray[1]
    };
}, {});
```

Reply:
Oh, I didn't consider the solution of the task from this angle.